x-pack/plugins/actions/server/builtin_action_types/case/utils.ts

```diff
@@ -1,111 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { flow, get } from 'lodash';
-
-import {
-  MapRecord,
-  TransformFieldsArgs,
-  Comment,
-  EntityInformation,
-  PipedField,
-  AnyParams,
-  PrepareFieldsForTransformArgs,
-} from './types';
-
-import { transformers } from './transformers';
-
-import { SUPPORTED_SOURCE_FIELDS } from './constants';
-
-export const normalizeMapping = (supportedFields: string[], mapping: MapRecord[]): MapRecord[] => {
-  // Prevent prototype pollution and remove unsupported fields
-  return mapping.filter(
-    (m) =>
-      m.source !== '__proto__' && m.target !== '__proto__' && supportedFields.includes(m.source)
-  );
-};
-
-export const buildMap = (mapping: MapRecord[]): Map<string, MapRecord> => {
-  return normalizeMapping(SUPPORTED_SOURCE_FIELDS, mapping).reduce((fieldsMap, field) => {
-    const { source, target, actionType } = field;
-    fieldsMap.set(source, { target, actionType });
-    fieldsMap.set(target, { target: source, actionType });
-    return fieldsMap;
-  }, new Map());
-};
-
-export const mapParams = <T extends {}>(params: T, mapping: Map<string, MapRecord>): AnyParams => {
-  return Object.keys(params).reduce((prev: AnyParams, curr: string): AnyParams => {
-    const field = mapping.get(curr);
-    if (field) {
-      prev[field.target] = get(params, curr);
-    }
-    return prev;
-  }, {});
-};
-
-export const prepareFieldsForTransformation = ({
-  externalCase,
-  mapping,
-  defaultPipes = ['informationCreated'],
-}: PrepareFieldsForTransformArgs): PipedField[] => {
-  return Object.keys(externalCase)
-    .filter((p) => mapping.get(p)?.actionType != null && mapping.get(p)?.actionType !== 'nothing')
-    .map((p) => {
-      const actionType = mapping.get(p)?.actionType ?? 'nothing';
-      return {
-        key: p,
-        value: externalCase[p],
-        actionType,
-        pipes: actionType === 'append' ? [...defaultPipes, 'append'] : defaultPipes,
-      };
-    });
-};
-
-export const transformFields = <
```

Comment:
Here @elastic/security-threat-hunting is the start of the migration we didn't catch